samples/build.gradle

```diff
@@ -19,7 +19,8 @@ android {
 }
 
 dependencies {
-  compile 'com.dlazaro66.qrcodereaderview:qrcodereaderview:2.0.1'
+  compile fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
I prefer to keep the example working with the latest library dependency. I know you have to change it locally in order to test the current library changes, but for the sake of consistency I strongly prefer to point to the maven release